packages/analytics/lib/index.d.ts

```diff
@@ -109,6 +109,10 @@ export namespace FirebaseAnalyticsTypes {
      * See https://github.com/invertase/react-native-firebase/issues/4578#issuecomment-771703420 for more information
      */
     price?: number;
+    /**
+     * Custom event parameters.
+     */
+    [key: string]: any;
```

Comment:
Only string and number are allowed here, please update. Worth updating information on custom parameters:

```
Parameter names can be up to 40 characters long and must start with an alphabetic character and contain only alphanumeric characters and underscores. String parameter values can be up to 100 characters long. The
"firebase_", "google_" and "ga_" prefixes are reserved and should not be used for parameter names.
```